processing/src/main/java/io/druid/segment/serde/LargeColumnSupportedComplexColumnSerializer.java

```diff
@@ -57,6 +58,7 @@ public LargeColumnSupportedComplexColumnSerializer(
     this.columnSize = columnSize;
   }
 
+  @PublicApi
```

Comment:
Since it's commonly used in implementations of ComplexMetricsSerde, which is an extension point.